### ISSUE TITLE
Fix plugin metrics: push test_errors and test_failures to SQ

### ISSUE THREAD

**Issue by andrei-epure-sonarsource** [SonarSource/sonar-dotnet#2764]
### Description

Currently, we seem to generate (see [UnitTestResultsTest](https://github.com/SonarSource/sonar-dotnet/blob/c6abd066b90df6d5b9f2fe484fb21d61ea404a30/its/src/test/java/com/sonar/it/csharp/UnitTestResultsTest.java#L56) ) 

-    tests
-    test_errors
-    test_failures
-    skipped_tests

However, we don't push them to SQ.

See [this community post](https://community.sonarsource.com/t/sonrqube-api-is-not-returning-test-success-density-for-c-net-application/13480)

From [this query call](https://next.sonarqube.com/sonarqube/api/measures/component?additionalFields=metrics&component=com.sonarsource.security%3Asonar-security-csharp-ucfg&metricKeys=tests,test_failures,test_errors) (sonarsource employees only)

```
    "measures": [
      {
        "metric": "test_errors",
        "value": "0",
        "bestValue": true
      },
      {
        "metric": "test_failures",
        "value": "0",
        "bestValue": true
      },
      {
        "metric": "tests",
        "value": "479"
      }
    ]
```

the metrics “test_errors” and “test_failures” are using best values, which means that they were not provided by the language analyzer.

because of this, other measures like `test_success_density` cannot get computed

### Expected behavior

Provide UT failure success metrics

### Actual behavior

Results are not pushed

### Known workarounds

None

### Related information

* SonarC# Version 8.0

**Comment by andrei-epure-sonarsource** [comment]
probably related to #886

**Comment by andrei-epure-sonarsource** [comment]
actually this is a duplicate of #886 , will close it

**Comment by andrei-epure-sonarsource** [comment]
marking this as won't fix, just like #886
